infrastructure/resource_group_level/modules/function-acbs.bicep

```diff
@@ -85,6 +88,7 @@ resource functionAcbs 'Microsoft.Web/sites@2022-09-01' = {
   properties: {
     httpsOnly: false
     serverFarmId: appServicePlanId
+    state: state
```

Comment:
Further to the previous comment, I dug deeper and found that actually the documentation is a combination of wrong and missing.

We can set the state, even though the REST interface and Azure docs say we can't.

I've raised and issue with the Azure CLI:
https://github.com/Azure/azure-rest-api-specs/issues/26097